src/com/itschool/session6/homework/ex3/ternaryOperator.java

```diff
@@ -0,0 +1,17 @@
+package com.itschool.session6.homework.ex3;
+
+import java.util.Scanner;
+
+public class ternaryOperator {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter your age");
+        int age = scanner.nextInt();
+        if(age >18){
+            System.out.println("I am adult");
+        } else if (age <18) {

```

Comment:
ok, but this is not ternary operator like the class name suggests. Can you refactor it to ternary operator? Also what if my age is 18, am I an adult or a child?